kingfisher_scrapy/commands/incrementaldatastore.py

```diff
@@ -0,0 +1,156 @@
+        """
+        cursor.execute(f'CREATE TABLE IF NOT EXISTS {spider_name} (data jsonb);')
+
+    def run_spider(self, last_date, spidercls):
+        kwargs = {'crawl_time': CRAWL_TIME}
+
+        # If there is data already in the database we only download data after the last release date
+        if last_date:
+            kwargs['from_date'] = self.from_date_formatted(last_date, spidercls.date_format)
+        self.crawler_process.crawl(spidercls, **kwargs)
+        self.crawler_process.start()
+
+    def get_data_from_directory(self, data_directory, json_data_path=''):
+        """
+        Yields items from jsons files in the given directory
+        :param data_directory directory from where read the json files
+        :param json_data_path the path from which read the data from the json file. By default the complete json file
+        """
+        for dir_entry in os.scandir(data_directory):
+            if dir_entry.name.endswith('.json'):
+                with open(dir_entry.path) as f:
+                    yield from ijson.items(f, json_data_path)
+
+    def json_to_csv(self, data, data_directory):
+        """
+        Receives an iterable of json dicts and save them into a data.csv file in the given directory
+        :param data an iterable of json dict
+        :param data_directory directory to where create the csv file
+        """
+        file_name = os.path.join(data_directory, 'data.csv')
+        with open(file_name, 'w') as csv_file:
+            csv_writer = csv.writer(csv_file)
+            for item in data:
+                data = json.dumps(item, default=util.default)
+                csv_writer.writerow([data])
+        return file_name
+
+    def run(self, args, opts):
+
+        if not self.settings['FILES_STORE'] or not os.getenv('KINGFISHER_COLLECT_DATABASE_URL'):
+            raise NotConfigured('You must set a FILES_STORE and KINGFISHER_COLLECT_DATABASE_URL')
+
+        spiders = self.crawler_process.spider_loader.list()
+        spider_name = opts.spider
+
+        if not spider_name or opts.spider not in spiders:
+            raise UsageError('A valid spider must be given.')
+
+        if not opts.db_schema:
+            raise UsageError('A database schema name must be given.')
+
+        spidercls = self.crawler_process.spider_loader.load(spider_name)
+
+        if 'record' in spidercls.data_type and opts.compile:
+            raise UsageError('The compile option can only be used with spiders that returns releases.')
+
+        # we disable kingfisher process extension
+        self.settings['EXTENSIONS']['kingfisher_scrapy.extensions.KingfisherProcessAPI'] = None
+
+        connection, cursor = self.database_setup(spider_name, opts.db_schema)
+
+        # gets the last data->>'date' that exists in the country table
+        cursor.execute(f"SELECT max(data->>'date') FROM {spider_name};")
+        last_date = cursor.fetchone()[0]
+
+        logger.info(f'Last release date found {last_date}')
```

Comment:
For clarity to the user, I prefer to log the command that we're simulating, e.g. 

```suggestion
        logger.info(f'Running: scrapy crawl -a from_date={kwargs["from_date"]} -a crawl_time={kwargs["crawl_time"]}')
```